### ISSUE TITLE
Enable users to sign-up and sign-in with github.

### ISSUE THREAD

**Issue by brunocroh** [brunocroh/bolhadev.chat#23]

**Comment by brunocroh** [comment]
I'm already working on it.

**Comment by brunooomelo** [comment]
I think it's a bad approach because English teachers don't have github.

**Comment by brunocroh** [comment]
> I think it's a bad approach because English teachers don't have github.

Yes you are right, I thinking here and we will have a alternative way to sign up with email/password as well.
